config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
```

Comment:
The Database adapter should be using `Postgresql` not `sqlite`. Please use `Postgres` as a database.

Reply:
Database set to postgres